### ISSUE TITLE
StandardShaderBall: Light interpretation variant

### ISSUE THREAD

**Issue by pablode** [usd-wg/assets#81]
Currently, light units and how values are interpreted vary between different renderers and Hydra-enabled applications (as can be seen in [this](https://github.com/anderslanglands/light_comparison) repository), causing the shader ball to most likely only render correctly in the environment it was authored in (Karma).

In order to get comparable results in other environments, we could add a variant that overrides the specific light properties, like intensities with values that have been calculated based on the original values. The logic could be based on above investigation and perhaps this [PR](https://projects.blender.org/blender/blender/pulls/109795) can provide additional information.

More specifically, it would also make sense to include (and perhaps default to) the [proposed update](https://github.com/PixarAnimationStudios/OpenUSD/pull/2758) of the UsdLux specification.

**Comment by crydalch** [comment]
Yes when the UsdLux proposals are accepted and light units/behaviors are better defined, we plan to update the scene to use those by default. It's probably not useful to keep the old, "wild-west" of light unit interpretations around once UsdLux is more rigorous.
